CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Change Log
 
+### Unreleased
+- Upgraded several dependencies
+- Upgraded to AGP 8.2.1
+- Upgraded to Kotlin 1.9.22
+- Upgraded to Gradle 8.5
+- MinSDK is now 23
```

Comment:
Breaking change, however the minSDK version we're on is SDK 26 so we should be able to pick this one up relatively easy.

Reply:
MinSDK version was updated due to build issues with tests. It's not absolutely necessary, but I didn't want to deal with the headache of supporting Android 4.x.